documentation/api/development.md

```diff
@@ -14,7 +14,11 @@ This section covers development using Docker. There are a number of Docker comma
 
 ### Setup
 
-Run `make init && make run-logs` to start the local containers. The application will be available at `http://localhost:8080` and API documentation at `http://localhost:8080/docs`.
+These scripts depend on having `postgresql` installed in your enviornment as a pre-requisite.
```

Comment:
A helpful link to the website will help users know what to do - and looks to be pretty thorough in handling all the various OS differences in installing (brew, apt, whatever WIndows is doing)

```suggestion
These scripts depend on having `postgresql` installed in your environment as a pre-requisite. Install instructions for your particular operating system can be found on the [Postgres website](https://www.postgresql.org/download/)
```